gradle/dependency-management/kotlin-versions.properties

```diff
@@ -1,2 +1,2 @@
 # Generated - Update by running `./gradlew updateKotlinVersions`
-latests=1.9.21,1.9.25,2.0.0,2.0.21,2.1.0,2.1.20
+latests=2.0.0,2.0.21,2.1.0,2.1.20
```

Comment:
These versions should be updated by changing the linked file and running the task

https://github.com/gradle/gradle/blob/7b9e3c8824191a3018c7c78e2e94c6c78c463805/build-logic/build-update-utils/src/main/kotlin/gradlebuild.update-versions.gradle.kts#L44-L49

It additionally updates our documentation to the new minimum version

Reply:
Oh, you mean like it says in the comment directly above the line I manually edited?  Yeah, I should probably do that, shouldn't I... 

Done.